testing/integration_test.sh

```diff
@@ -26,7 +26,7 @@ cp -R "testing" "$GOPATH/src/proftest"
 cd "$GOPATH/src/proftest"
 go get -t -tags=integration .
 if [ "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" == "" ]; then
-  go test -timeout=30m -parallel=3 -tags=integration -run TestAgentIntegration -commit="$COMMIT" -branch="$BRANCH" -repo="$REPO"
+  go test -timeout=30m -parallel=4 -tags=integration -run TestAgentIntegration -commit="$COMMIT" -branch="$BRANCH" -repo="$REPO"
```

Comment:
Not super related to this PR, but: https://golang.org/cmd/go/#hdr-Testing_flags says "The value of this flag is the maximum number of tests to run simultaneously; by default, it is set to the value of GOMAXPROCS." To avoid having to maintain the value here in sync with the number of tests, I would add something like `runtime.GOMAXPROCS(len(testcases))` in the test function instead.

Reply:
@nolanmar511 Did you see this comment?

Reply:
I switched to using this.